lib/grape/validations/autoload.rb

```diff
@@ -0,0 +1,52 @@
+module Grape
+  module Validations
+    class Autoload
+      DEFAULT_PATH = %w(api validators)
```

Comment:
I agree. What about the autoload tries to find the constant in grape validators directory `lib/grape/validations/validators/custom.rb`?

Client would need to put their custom validators in `lib/grape/validations/validators/` which sounds reasonable to me.